elliottlib/bzutil.py

```diff
@@ -100,11 +100,21 @@ def __init__(self, bug_obj: Issue):
         self.component = self.bug.fields.components[0].name
         self.status = self.bug.fields.status.name
         self.summary = self.bug.fields.summary
+        self.blocks = self.get_blocks()
+        self.keywords = self.bug.fields.labels
```

Comment:
I'm not sure about this change.. keywords are going to go away when bugzilla goes away.
But for now I think it's okay